domain/user/service/service.go

```diff
@@ -71,7 +77,7 @@ const (
 	// key.
 	activationKeyLength = 32
 
-	// usernameValidationRegex is the regex used to validate that user names are
+	// usernameValidationRegex is the regex used to validate that usernames are
```

Comment:
Not sure why these words were joined. While `username` might be a common variable name, plain English prose should be preferred for comments.

Reply:
Fair enough, looks like this is my misprint